src/common/AdaptiveTree.cpp

```diff
@@ -21,6 +21,7 @@
 #endif
 
 #include <algorithm>
+// #include <cassert>
```

Comment:
Dead code?

Reply:
intentional for debug, but is not part of this PR